app/models/edm/web_resource.rb

```diff
@@ -87,6 +88,8 @@ class WebResource
       application/pdf
     ).freeze
 
+    MAX_MEDIA_SIZE = 50000000
```

Comment:
Could we write the number here as: `50.megabytes`

Reply:
this isn't quite the same number, so before sending it to the view I'm doing some additional processing to it.